src/WebJobs.Script/Extensions/TraceWriterExtensions.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Azure.WebJobs.Host;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public static class TraceWriterExtensions
```

Comment:
We have a `TraceWriterExtensions` class that was introduced with the conditional trace writer:
https://github.com/Azure/azure-webjobs-sdk-script/blob/dev/src/WebJobs.Script/Diagnostics/TraceWriterExtensions.cs
We should merge those two. I like having it under `Diagnostics`, but don't feel strongly about it.